sound/soc/sof/topology.c

```diff
@@ -954,6 +962,12 @@ static int sof_control_load_volume(struct snd_soc_component *scomp,
 	}
 
 out:
+	/* set up possible led control from mixer private data */
+	scontrol->led_ctl.use_led = 0;
```

Comment:
is this init required or is this already initialized during the scontrol creation?

Reply:
yes you're right, it is done with kzalloc, so removing.